sklearn/discriminant_analysis.py

```diff
@@ -514,8 +514,7 @@ def _solve_svd(self, X, y):
         self.intercept_ -= np.dot(self.xbar_, self.coef_.T)
 
     def fit(self, X, y):
-        """Fit LinearDiscriminantAnalysis model according to the given
-           training data and parameters.
+        """Fit model according to the given training data and parameters.
```

Comment:
Maybe

```suggestion
        """Fit the Linear Discriminant Analysis model.
```
?

"Fit model according to the given training data and parameters." is a bit of a tautology.

Reply:
Good point ;). My suggestion was bad. I am merging this.